### DISCUSSION TITLE
English -> English placeholder translation with English as source language

### DISCUSSION THREAD

**Discussion by luebbe** [WeblateOrg/weblate#8315] [Q&A] [answered]
Hello,
in a project that we want to move to weblate, the source language is English. We are using .po(t) files. Most of the msgids are normal ui strings, but some are placeholder IDs like `msgid "PLACEHOLDER_TEXT"`.
In order to get these displayed on the UI, there are .po files en->de, en->xxx, ... **and en->en**.
These .po files expand the placeholder to the full text in the given language. When we now initialize weblate with the existing .pot and .po files, it complains that the translation component is not correct, because it also contains a translation for the source language (en->en).
Which would be the best way to resolve this issue?
- Is there something like a "pseudo" language, which could be used as target, when an English UI is desired (like en_xx)?
- Shall we replace the placeholders in the sources with the full text and get rid of them entirely?
- Other options?

**Comment by nijel** [comment, marked as answer]
You can use `English (Developer)` as source language, see https://docs.weblate.org/en/latest/faq.html#what-does-mean-there-are-more-files-for-the-single-language-en for more info.
